### ISSUE TITLE
LQIP with folder in S3

### ISSUE THREAD

**Issue by M-Blobby** [imagekit-developer/imagekit-angular#16]
Hi,

I'm trying to add LQIP function, unfortunately without success.
My pictures are stored in AWS S3 in different folders :
- projects/XX.jpg
- covers/XX.jpg

My component.html :
```html
<ik-image [path]="path" [lqip]="lqip"></ik-image>
```

My component.ts : 
```javascript
lqip = { active: true, quality: 5 };
```

In the network tab I catch this url :
**https://ik.imagekit.io/token/tr:q-5/projects**
Why the end of the path is missing ?

Without lqip I can see my image :
https://ik.imagekit.io/token/projects/13/XX.jpg?ik-sdk-version=angular-1.0.2

Other question : There is a way to add blur on lqip ?

**Comment by justjoeyuk** [comment]
It looks like this lib is much less loved that the React one lmao

**Comment by imagekitio** [comment]
Fixed in version 2.0.0.
